www/front_src/src/Resources/Details/index.test.tsx

```diff
@@ -1801,4 +1806,37 @@ describe(Details, () => {
       expect(getByText(email)).toBeInTheDocument();
     });
   });
+
+  it('calls the download timeline endpoint when the Timeline tab is selected and the "Export to CSV button" is clicked', async () => {
+    mockedAxios.get.mockResolvedValueOnce({ data: retrievedDetails });
+    mockedAxios.get.mockResolvedValueOnce({ data: retrievedTimeline });
+
+    const mockedOpen = jest.fn();
+    window.open = mockedOpen;
+
+    setUrlQueryParameters([
+      {
+        name: 'details',
+        value: serviceDetailsTimelineUrlParameters,
+      },
+    ]);
+
+    const start = '2020-01-20T06:00:00.000Z';
+
+    const { getByTestId } = renderDetails();
+
+    await waitFor(() => {
+      expect(mockedAxios.get).toHaveBeenCalledTimes(3);
+    });
+
+    act(() => {
+      fireEvent.click(getByTestId(labelExportToCSV));
+    });
```

Comment:
```suggestion
      fireEvent.click(getByTestId(labelExportToCSV));
```
fireEvent handle the `act`

Reply:
ok thank's